pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -54,6 +58,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	return nil
 }
 
+var engine engineInfo
+
+var restConfig *rest.Config
+var clientset *kubernetes.Clientset
```

Comment:
Instead of defining the global config and clientset can you define inside the function.

Reply:
If i will define config and clientset locally it will increase no. of  arguments in a function.

Reply:
@ksatchit What you suggest, Should we go ahead and define config and clientset globally?